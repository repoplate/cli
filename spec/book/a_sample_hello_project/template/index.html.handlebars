<html>
  <body>
    <a href="./{{greeting}}.html">:)</a>
  </body>
</html>
