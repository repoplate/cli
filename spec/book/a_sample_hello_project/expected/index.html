<html>
  <body>
    <a href="./hello.html">:)</a>
  </body>
</html>
